LeaderboardBackend/Models/Entities/User.cs

```diff
@@ -1,14 +1,28 @@
 using System.ComponentModel.DataAnnotations;
-using System.ComponentModel.DataAnnotations.Schema;
 using System.Text.Json.Serialization;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
 
 namespace LeaderboardBackend.Models.Entities;
 
+public enum UserRole : byte
+{
+    None = 0, // should not be used
```

Comment:
Yeah, I think this should be safe to remove.

Reply:
I like enums to start at 1 rather than zero because it means it was explicitly chosen, and the default value of numeric types will be invalid.
We can remove the None line and do `Registered = 1`  to achieve this though.